Events.lua

```diff
@@ -1350,22 +1350,29 @@ local function UNIT_POWER_FREQUENT( event, unit, power )
             state.focus.last_tick = now
         end
 
-    elseif power == "ENERGY" and rawget( state, "energy" ) then
+    elseif power == "ENERGY" and rawget(state, "energy") then
         local now = GetTime()
         local elapsed = state.energy.last_tick > 0 and now - state.energy.last_tick or 2
-        local current_energy = UnitPower( "player", Enum.PowerType.Energy )
+        local current_energy = UnitPower("player", Enum.PowerType.Energy)
         local tick_energy = current_energy - lastObservedEnergy
-        if (tick_energy == 20 or current_energy == state.energy.max) and elapsed >= 1.9 and state.action.cat_form.realCast ~= state.now then
+    
+   
+        if (tick_energy == 20 or current_energy == state.energy.max) and elapsed >= 1.9 then
```

Comment:
Please re-add the cat_form check, but you can encapsulate in parens with a short-circuit check e.g.

((not state.action.cat_form) or state.action.cat_form.realCast ~= state.now)